Makefile

```diff
@@ -373,7 +372,7 @@ clean: clean-debug
 .PHONY: start
 start:
 	docker version
-	$(call run-in-test-server,make ARGOCD_PROCFILE=test/container/Procfile start-local)
+	$(call run-in-test-server,make ARGOCD_PROCFILE=test/container/Procfile start-local ARGOCD_START=${ARGOCD_START})
```

Comment:
Added ability to run components independently. E.g. `make start ARGOCD_START=api-server`